tests/Doctrine/Tests/Common/Reflection/RuntimePublicReflectionPropertyTest.php

```diff
@@ -78,7 +78,7 @@ class RuntimePublicReflectionPropertyTestProxyMock implements Proxy
     private $initializer     = null;
 
     /**
-     * @var \Closure|null
+     * @var \Closure|bool
```

Comment:
[That states `bool`](https://github.com/doctrine/common/blob/56fff715d14801b8883eb7e981f010e3c0587f6c/lib/Doctrine/Common/Persistence/Proxy.php#L56), but `__isInitialized` returns `bool|\Closure`